docs/roman/dq_init/description.rst

```diff
@@ -35,10 +35,10 @@ WFI_WIM_ACQ, WFI_WIM_TRACK, WFI_WSM_ACQ1, WFI_WSM_ACQ2, WFI_WSM_TRACK.
 Conversion from Level 1 uncal files
 -----------------------------------
 
-The output of the initialization step is the RampModel. This is the form of the
-data used throughout most of the exposure pipeline steps. For the most part, the
-meta information between the input raw, or "uncal", model and the ramp model is
-complete.
+The DQ init takes an input raw "uncal" file and outputs a intermediate
+"ramp" model (RampModel) that is the input for all following steps
+until ramp fitting.  For the most part, the meta information between
+the input raw, or "uncal", model and the ramp model is identical.
```

Comment:
I'm not really sure what this paragraph is trying to say. For instance what is an "intermediate" ramp model?
I think you're trying to say the the initial ramp model from SDF has a data array that are the differences described below? I'd just say that instead of implying we have different ramp models. 

Should we have an attribute that indicates if the data values are differences or absolute values?

Reply:
I'll delete "intermediate".  This model is intermediate for me in the sense that we never write this to disk in operations.  The only 3D object most users are likely to see is the uncal model.